<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rebus Mekanizması / Görselden Kelimeye</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
   background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>420</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #114abd;
	color: white;
    border-style: solid;
    border-radius: 5;
    
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:hover {
   background-color: #286fff;
}

QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
}
</string>
    </property>
    <property name="text">
     <string>TAHMİN ET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeRebusBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>224</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    border-style: solid;
    border-radius: 5;
    
    min-width: 9ex;
    min-height: 2.5ex;
}

QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/changeRebusIcon.png</normaloff>images/changeRebusIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>307</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4a4a4a;&quot;&gt;Girilecek Kelime&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>530</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
